cpu-miner.c

```diff
@@ -595,7 +595,7 @@ static bool submit_upstream_work(CURL *curl, struct work *work) {
 
             case ALGO_K12:
             default:
-                k12_hash(hash, work->data, work->dlen);
```

Comment:
need to keep data len, yeah it's 76+4.
the test function "reference inputs" length are not 80